File: python/program/Human.py
class Human:
    def __init__(self, NIK="", nama="", jenis_kelamin=""):  # constractor
        self.NIK = NIK
        self.nama = nama
        self.jenis_kelamin = jenis_kelamin

    # Getter and Setter

    def set_NIK(self, NIK):
        self.NIK = NIK

    def set_Nama(self, nama):
        self.nama = nama

    def set_Jenis_kelamin(self, jenis_kelamin):
        self.jenis_kelamin = jenis_kelamin

    def get_NIK(self):
        return self.NIK

    def get_Nama(self):
        return self.nama

    def get_Jenis_kelamin(self):
        return self.jenis_kelamin

File: CPP/PROGRAM/Human.cpp
#include <bits/stdc++.h>

using namespace std;

class Human
{
private: // private atribut
    string NIK;
    string nama;
    string jenis_kelamin;

public:
    Human() // CONSTRUCTOR
    {
    }

    void setNIK(string b) // SET NIK
    {
        NIK = b;
    }
    void setNama(string c) // SET nama
    {
        nama = c;
    }
    void setJenis_kelamin(string d) // SET jenis_kelamin
    {
        jenis_kelamin = d;
    }

    string getNIK() // GET NIK
    {
        return NIK;
    }
    string getNama() // GET nama
    {

        return nama;
    }
    string getJenis_kelamin() // GET jenis_kelamin
    {
        return jenis_kelamin;
    }
    ~Human() // DESTRUCTOR
    {
    }
};